src/components/Header/index.tsx

```diff
@@ -20,24 +20,39 @@ import { AppContext } from '../../contexts/providers/index'
 import { AppDispatch, ComponentActions } from '../../contexts/providers/reducer'
 import Dropdown from './Dropdown'
 
+enum LinkType {
+  Inner,
+  Outer,
+}
+
 const Menus = () => {
   const [t] = useTranslation()
   const MenuDatas = useMemo(() => {
     return [
       {
+        type: LinkType.Inner,
         name: t('navbar.charts'),
         url: '/charts',
       },
+      {
+        type: LinkType.Outer,
+        name: t('navbar.faucet'),
+        url: 'https://faucet.nervos.org',
```

Comment:
Can we open a separate window to Faucet?

Reply:
OK. I see. Maybe it is a better way.